apps/passport-client/components/shared/PCDArgs.tsx

```diff
@@ -379,6 +386,58 @@ export function ObjectArgInput({
   );
 }
 
+function RevealArgInput({
```

Comment:
i think it would be good to rename this to something more general - eg 'togglelist' - something that lets you toggle a bunch of values independently. that way it could be reused by other PCDs as well, in case they need a toggle list that doesn't necessarily toggle things that mean 'reveal or not reveal'